llvm/test/Transforms/OpenMP/deduplication_soundness.ll

```diff
@@ -0,0 +1,70 @@
+; RUN: opt -passes=openmp-opt-cgscc -S < %s | FileCheck %s
+
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+
+declare i32 @printf(ptr noundef, ...)
```

Comment:
Just make this some external call instead of printf.

Reply:
Done!

Let me know if any further changes may be required